pkg/types/ccipocr3/config.go

```diff
@@ -78,3 +62,13 @@ type ObserverInfo struct {
 	// Reads define the chains that the current node can read from.
 	Reads []ChainSelector `json:"reads"`
 }
+
+// ChainConfig will live on the home chain and will be used to update chain configuration like F value and supported nodes dynamically.
+type ChainConfig struct {
```

Comment:
Is this type also unused from this module?

Reply:
Yes. Will move it now 👍